src/gui/sdlmain.cpp

```diff
@@ -4311,20 +4314,10 @@ void Config_Add_SDL() {
 	mouse_control_help += mouse_control_defaults;
 	Pmulti->Set_help(mouse_control_help);
 
-	Pmulti = sdl_sec->AddMultiVal("sensitivity", always, ",");
-	Pmulti->Set_help("Mouse sensitivity. The optional second parameter specifies vertical sensitivity\n"
-	                 "(e.g. 100,-50).");
-	Pmulti->SetValue("100");
-	Pint = Pmulti->GetSection()->Add_int("xsens", always,100);
-	Pint->SetMinMax(-1000,1000);
-	Pint = Pmulti->GetSection()->Add_int("ysens", always,100);
-	Pint->SetMinMax(-1000,1000);
-
-	pbool = sdl_sec->Add_bool("raw_mouse_input", on_start, false);
-	pbool->Set_help(
-	        "Enable this setting to bypass your operating system's mouse\n"
-	        "acceleration and sensitivity settings. This works in\n"
-	        "fullscreen or when the mouse is captured in window mode.");
+	Pmulti = sdl_sec->AddMultiVal("sensitivity", deprecated, ",");
+	Pmulti->Set_help("Moved [mouse] section; see documentation as values have different meaning now.");
```

Comment:
-> "Moved **to** [mouse] section."

And (I think?) we can drop the "see documentation as values have different meaning now." portion?

Reply:
Right, changed.